src/backends/policy_engine/souffle/check_policy_compliance.cc

```diff
@@ -61,6 +63,51 @@ absl::StatusOr<std::string> ReadFileContents(std::filesystem::path file_path) {
   return string_stream.str();
 }
 
+enum class ReturnCode : uint8_t { PASS = 0, FAIL, ERROR };
```

Comment:
For the sake of exit code stability, you may want to make every enumeration have an explicit value. Imagine if we added a case like this:

```C++
enum class ReturnCode : uint8_t { PASS = 0, PARTIAL_SUCCESS, FAIL, ERROR };
```

Now the `FAIL` and `ERROR` cases would silently change their value, which could cause issues if we had hardcoded them into tests outside of this file (a pretty reasonable thing to do for exit codes).